server/src/main/java/io/seata/server/lock/memory/MemoryLocker.java

```diff
@@ -71,7 +71,7 @@ public boolean acquireLock(List<RowLock> rowLocks) {
         String resourceId = branchSession.getResourceId();
         long transactionId = branchSession.getTransactionId();
 
-        ConcurrentHashMap<Map<String, Long>, Set<String>> bucketHolder = branchSession.getLockHolder();
+        ConcurrentHashMap<ConcurrentHashMap<String, Long>, Set<String>> bucketHolder = branchSession.getLockHolder();
```

Comment:
Do we have to change the type declaration to `ConcurrentHashMap`?

Reply:
I think it is appropriate to use ConcurrentHashMap to declare bucket map, it could help next guy understand why it's thread safe.

Reply:
Ok, it's helpful.